src/snowflake/snowpark/dataframe_reader.py

```diff
@@ -40,6 +40,20 @@
 else:
     from collections.abc import Iterable
 
+# We allow some common aliases to be used
+option_aliases = {
+    "HEADER": ("SKIP_HEADER", lambda val: 1 if val else 0),
```

Comment:
```suggestion
OPTION_ALIASES = {
    "HEADER": ("SKIP_HEADER", lambda val: 1 if val else 0),
```
nit: in all our files, constants are declared with all caps